packages/app/plugin/tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "@tsconfig/node12/tsconfig.json",
```

Comment:
Out of curiosity, why this specific tsconfig node12 bit? Smells like maintainer-burden for the future node n+1 release etc

Reply:
I added this to avoid adding much heavier and not-really-necessary [`expo-module-scripts`](https://www.npmjs.com/package/expo-module-scripts) dependency.

My `tsconfig` should extend the [`expo-module-scripts/tsconfig.plugin.json`](https://github.com/expo/expo/blob/master/packages/expo-module-scripts/tsconfig.plugin.json), but when we see that tsconfig, it only extends the `node12` tsconfig, so why not use it directly 😄

Reply:
Interesting - I just spidered out and examined the whole `@tsconfig` thing. Interesting idea. This will be a maintenance burden syncing/updating over time, that's unavoidable as there must be some future state where ES2022 is the minimum required javascript target or similar, but it appears it may be a good maintenance burden in that it actually abstracts away other pain points in keeping tsconfigs in sync + valid. May adopt it in other areas. Resolved, in my mind, thanks